### PR TITLE
[circle-quantizer] change `profiling_data_gen` option name

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit renames `profiling_data_gen` to `generate_profile_data`.

Related: https://github.com/Samsung/ONE/pull/6250#discussion_r591017494
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

**Review by jinevening — DISMISSED** [review]
LGTM

Nit: gpd would be better

File: compiler/circle-quantizer/src/CircleQuantizer.cpp [hunk 1]
```diff
@@ -64,7 +64,7 @@ int entry(int argc, char **argv)
   const std::string qwmm = "--quantize_with_minmax";
   const std::string rq = "--requantize";
 
-  const std::string pdg = "--profiling_data_gen";
+  const std::string pdg = "--generate_profile_data";
```
Comment (seanshpark) [inline compiler/circle-quantizer/src/CircleQuantizer.cpp:67]: `pdg` -> `gpd` :)

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by jinevening — APPROVED** [review]
LGTM Thank you

**Pull request merged**